jjb/mkck/mkck-trigger.yaml

```diff
@@ -508,7 +524,9 @@
                 - snippets/mkck-target-properties.bash
         - inject:
             properties-file: 'target.properties'
-        - shell: "git clone https://github.com/suse/sesci -b $CI_BRANCH"
+        - git-clone:
+            git_url: "https://github.com/suse/sesci"
+            git_branch: "$CI_BRANCH"
```

Comment:
Looks good, but I'm not sure that using macro for one-liner is efficient, especially :-) if to replace it with more lines of code

Reply:
True. do you want me to remove that commit?

Reply:
yes, please

Reply:
@kshtsk done